pysatMadrigal/tests/test_methods_gnss.py

```diff
@@ -31,3 +39,49 @@ def test_ref_output(self, func, comp_str, in_args):
         self.out = getattr(gnss, func)(*in_args)
         assert self.out.find(comp_str) >= 0
         return
+
+
+class TestGNSSBadLoad(object):
+    """Test GNSS load warnings and errors."""
+
+    def setup_method(self):
+        """Run before every method to create a clean testing setup."""
+        self.bad_fnames = ['los_20230101.simple.gz', 'los_20230102.netCDF4']
+        return
+
+    def teardown_method(self):
+        """Run after every method to clean up previous testing."""
+        del self.bad_fnames
+        return
+
+    def test_bad_file_type_warning(self, caplog):
+        """Test logger warning for unsupported file types loading LoS data."""
+
+        # Get the output and raise the logging warning
+        with caplog.at_level(logging.WARN, logger='pysat'):
+            gnss.load_los(self.bad_fnames, "site", "zzon")
+
+        # Test the logger warning
```

Comment:
May want to use the pysat.testing warnings methods.

Reply:
Since you've identified places, recommend creating an issue and adding TODO statements at the locations you found.